pkg/machines/components/create-vm-dialog/createVmDialog.less

```diff
@@ -15,3 +15,11 @@
 #system-select .dropdown-menu {
     max-height: ~"calc(90vh - 50px - 10em)";
 }
+
+
+/* We use form-group in divs for inline validaton of dialog values
+ * and we don't need extra margins there.
+ */
+#create-vm-dialog div.form-group {
+    margin-bottom: 0 !important;
```

Comment:
If you can figure out a way to not use !important here, I think that would be healthier for the code long term. Maybe by specifying the selector a bit closer?

Reply:
@garrett any good ideas for how to solve this without the !important?

Reply:
I fixed by just specifying a more specific selector.

Reply:
Awesome, thanks!